src/components/common/LabeledSelect.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import styled from 'styled-components'
+import Select from 'react-select'
+import find from "lodash/find"
+
+const getStringValues = (value) => {
+    return value == null ? [] : value.map((item) => item.value);
+}
+
+const LabeledSelect = ({label, value, options, onChange, ...rest}) => (
+    <Wrapper>
+        <label>{label}</label>
+        <StyledSelect
+            placeholder="Any"
+            value={value ? find(options, { value }) : null}
+            onChange={(value) => onChange( rest.isMulti ? getStringValues(value) : value)}
```

Comment:
isMulti

Reply:
onChange should call a function that handles the isMulti logic and has a reference to a parent function to update state (if needed)

Reply:
done